test/functional/api/vim_spec.lua

```diff
@@ -1036,5 +1036,33 @@ describe('api', function()
       eq(expected, nvim("list_uis"))
     end)
   end)
-
+  describe('nvim_get_keyboard_mods_state', function()
+    it('on linux either xlib11 is available or not', function()
+      if os_name() == 'unix' then
+        local status, rv = pcall(nvim, "get_keyboard_mods_state")
```

Comment:
Ah, and there is such a thing as `meths` dictionary-like object. You don’t need `nvim`, you can use `meth.get_keyboard_mods_state` function. I use that in all new tests as it is less verbose.